src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -172,6 +181,91 @@ public static void createTempPartitionedSourceBQTable() throws IOException, Inte
     BeforeActions.scenario.write("BQ Source Table " + bqSourceTable + " created successfully");
   }
 
+  /**
+   * Create BigQuery table with different list of dataypes (transaction_info BOOL, transaction_num BYTES,
+   * transaction_uid STRING, transaction_date DATE, transaction_dt DATETIME, transaction_time TIME, latitude FLOAT64,
+   * unique_key INT64, business_ratio NUMERIC, updated_on TIMESTAMP, parent ARRAY<STRING>,
+   * inputs STRUCT<input_script_bytes BYTES, input_script_string STRING, input_sequence_number INT64>)
+   * containing random testdata.
+   */
+  @Before(order = 1, value = "@BQ_SOURCE_DATATYPE_TEST")
+  public static void createSourceBQTableWithDifferentDataTypes() throws IOException, InterruptedException {
+    createSourceBQTableWithQueries(PluginPropertyUtils.pluginProp("bqCreateTableQueryFile"),
+            PluginPropertyUtils.pluginProp("bqInsertDataQueryFile"));
```

Comment:
Indent looks incorrect. Please reformat all files in the PR using this settings file - https://cdap.atlassian.net/wiki/spaces/CE/pages/1594098619/Coding+Standard

Reply:
Changes done.